tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -269,9 +269,12 @@ user deletes editable accordion section content block
     user clicks button  Remove block  ${block}
     user clicks button  Confirm
 
-user gets meta guidance data file content editor
-    [Arguments]   ${accordion_heading}
+user enters text into meta guidance data file content editor
+    [Arguments]   ${accordion_heading}   ${text}
     ${accordion}=  user gets accordion section content element  ${accordion_heading}
     user waits until parent contains element  ${accordion}   xpath://*[@data-testid="Content"]//*[@role="textbox"]
     ${editor}=  get child element  ${accordion}  xpath://*[@data-testid="Content"]//*[@role="textbox"]
-    [Return]  ${editor}
+    user checks page does not contain testid   fileGuidanceContent-focused
+    user clicks element  ${editor}
+    user waits until page contains testid   fileGuidanceContent-focused
```

Comment:
I did a lot of umming and ahhing about how to wait for the content editor to be ready to receive text. In the end, I decided this was the best option.

Reply:
Has there been problems with the editor being ready to receive text? Running it locally, it seems to run fine

Reply:
It's been failing intermittently - I'm not 100%, but I think it's only been happening on the pipeline. Anyway, I'm hoping this fixes it.

Reply:
Cool, we'll go with it then. Wish the pipeline wouldn't randomly fail on stuff like this though!

Reply:
I have wondered if we had laptops that are the same specs as the Azure agents we might be more likely to recreate this stuff locally - almost always pipeline only failures seem to be timing related.

Reply:
Hah good luck running all of EES on just 2 cores!